### DISCUSSION TITLE
How can I backup vaultwardem data for changing server?

### DISCUSSION THREAD

**Discussion by mizzunet** [dani-garcia/vaultwarden#1750] [Q&A] [answered]
I hosted vaultwarden in a server. I have another server. I want to zipped /vw-data and unzipped it and put  /vw-data the second server, then ran 
docker pull vaultwarden/server:latest
docker run -d --name vaultwarden -v /vw-data/:/data/ -p 80:80 vaultwarden/server:latest
But the account I used to login with older server, says password or username incorrect, how do I backup it properly?

**Comment by RealOrangeOne** [comment, marked as answer]
https://github.com/dani-garcia/vaultwarden/wiki/Backing-up-your-vault

**Reply by mizzunet** [reply]
Thanks a lot

**Reply by mizzunet** [reply]
I tested
1. Deployed vaultwarden
2. Created an account.
3.backed up the `sqlite by sqlite3 /vw-data/db.sqlite3 /db.sqlite3
`
4. Killed the docker container and deleted `/vw-data`
5. Deployed vaultwarden again
6. Ran `rm /vw-data/db.sqlite3` and `mv  /db.sqlite3 /vw-data`
7. Tried to login the account I created before backing up, it fails :(

What could be the issue? Am I missing anything here?
